packages/sync/README.md

```diff
@@ -17,3 +17,140 @@ To enable debug please execute in console
 Some certain features can be enabled separately
 
 `localStorage.debug = 'AeroGearSync:OfflineMutations*'`
+
+# Importing the package
+```javascript
+import {
+  createClient,
+  strategies
+} from '@aerogear/datasync-js';
+```
+
+# Configuration
+To provide custom configuration to the client, the following options are available. If you wish, these are also available by using the `DataSyncConfig` interface from the SDK.
+
+```javascript
+declare var window: any;
+
+let config: DataSyncConfig = {
```

Comment:
Kinda pointless to have those without explanation. So we could just point people to DataSyncConfig file